### DISCUSSION TITLE
Make elementor interface language selector different from the Client language

### DISCUSSION THREAD

**Discussion by urigross** [elementor/elementor#22020] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I need to output a website in Hebrew language to the client - For that I need to change the Elementor and the whole Wordpress to Hebrew as well although I like working and as the whole world, work in English.

Attached an image which Gallery output is Hebrew ( "הכל" ) category. For that I needed to change the whole Worpress setting to Hebrew.
![image](https://user-images.githubusercontent.com/37923167/231550676-7d165efa-e0a6-447c-a61c-3125a30323ab.png)

### Describe the solution you'd like

The Elementor Hebrew Output to the website selector will be independent so I could keep the Wordpress site setting in English and the Output will be the way I will choose.

### Describe alternatives you've considered

Independent selector for elements Language output.

**Comment by arielk** [comment]
See here: https://elementor.com/help/set-editor-language/
